src/view-components/tabbed-view/tabbed-view.component.tsx

```diff
@@ -12,14 +12,15 @@ import {
 
 import styles from "./tabbed-view.css";
 import { getView, ViewType } from "../view-utils";
+import { NavbarType } from "../../root.component";
```

Comment:
Same goes for NavbarType and ViewType. What are they? Are they models?

Reply:
What do you mean by model? I'm only familiar with types and interfaces.